### ISSUE TITLE
Error when using Show-InstallationWelcome with specific parameters

### ISSUE THREAD

**Issue by Nabsku** [PSAppDeployToolkit/PSAppDeployToolkit#305]
Hey guys, 

having an issue here. When using Show-InstallationWelcome -CloseApps 'Acrobat' -BlockExecution the Toolkit (AppDeployToolkitMain.ps1 at line 5178) crashes after the Installation, when trying to clean up the ScheduledTask it created (or tried to create). 

Error being: **A parameter cannot be found that matches parameter name 'ContinueOnError'.**

This only happens when I use the BlockExecution with CloseApps 'Acrobat'. I've temporarily solved the issue with replacing the -ContinueOnError with -ErrorAction Continue but I don't know what I broke with that in turn.

If there are any Information you guys need, I can provide them.

**Comment by seanlillis** [comment]
Hi there, if you could provide the log file from the failed install that would be great.

**Comment by Nabsku** [comment]
You can find the logfile attached below!

Thanks for your answer @seanlillis 

[Logfile of Acrobat Install](https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/files/1367603/Adobe_AcrobatDCProfessional_15.006.30033_x86_DE_02_PSAppDeployToolkit_Install.log)

**Comment by seanlillis** [comment]
Thanks for the log. I can't reproduce this. Would you mind sending a copy of the AppDeployToolkitMain.ps1 script?

**Comment by Nabsku** [comment]
Thank you very much for that change. I've attached our current AppDeployToolkitMain.ps1 and Deploy-Application.ps1. 

Do note:  I've changed the Unblock-AppExecution from -ContinueOnError to -ErrorAction SilentlyContinue.

Also, I've redacted some Information about our company.

[Temp.zip](https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/files/1401010/Temp.zip)

**Comment by seanlillis** [comment]
The Deploy-Application.ps1 is showing as 0KB.

**Comment by Nabsku** [comment]
I have no clue how that happened. 

Excuse that.

**Comment by Nabsku** [comment]
I have no clue on how that happened.

Heres a working copy!
[Files.zip](https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/files/1465819/Files.zip)

**Comment by seanlillis** [comment]
I was able to make a few modifications to your script in order to simulate a test run, and I cannot reproduce the issue. It's a really strange one and I'm wondering if there is something in your script modifications causing the issue.

You could try specifying the exact .exe processes to close rather than using 'Acrobat'. Also, I notice that you have modified the below line in Deploy-Application.ps1, I would suggest changing the ' to " in order to pick up the variables. 

 [string]$deployAppScriptFriendlyName = '$appVendor $appName $appVersion $appPlattform $appArch'

**Comment by luki1412** [comment]
This version of the toolkit is very outdated, please use the latest one and create a new issue if the problem persists.
